alchemy/src/cloudflare/email-routing.ts

```diff
@@ -0,0 +1,224 @@
+import type { Context } from "../context.ts";
+import { Resource } from "../resource.ts";
+import { handleApiError } from "./api-error.ts";
+import {
+  type CloudflareApi,
+  type CloudflareApiOptions,
+  createCloudflareApi,
+} from "./api.ts";
+import type { CloudflareResponse } from "./response.ts";
+import type { Zone } from "./zone.ts";
+
+/**
+ * Cloudflare Email Routing response format
+ */
+interface CloudflareEmailRouting {
+  enabled: boolean;
+  name: string;
+  created: string;
+  modified: string;
+  tag: string;
+}
+
+/**
+ * Properties for configuring email routing on a zone
+ */
+export interface EmailRoutingProps extends CloudflareApiOptions {
+  /**
+   * Zone ID or Zone resource where email routing will be configured
+   */
+  zone: string | Zone;
+
+  /**
+   * Whether email routing should be enabled
+   * 
+   * @default true
+   */
+  enabled?: boolean;
+
+  /**
+   * Skip the DNS setup wizard when enabling email routing
+   * 
+   * @default false
+   */
+  skipWizard?: boolean;
+}
+
+/**
+ * Email routing configuration for a Cloudflare zone
+ */
+export interface EmailRouting extends Resource<"cloudflare::EmailRouting"> {
+  /**
+   * Zone ID where email routing is configured
+   */
+  zoneId: string;
+
+  /**
+   * Whether email routing is enabled
+   */
+  enabled: boolean;
+
+  /**
+   * Zone name
+   */
+  name: string;
+
+  /**
+   * When email routing was created
+   */
+  created: string;
+
+  /**
+   * When email routing was last modified
+   */
+  modified: string;
+
+  /**
+   * Zone tag
+   */
+  tag: string;
+}
+
+/**
+ * Configures email routing for a Cloudflare zone, allowing emails sent to the zone's domain
+ * to be routed according to defined rules.
+ *
+ * @example
+ * ## Enable email routing for a zone
+ *
+ * Enable email routing for your domain to start receiving and routing emails.
+ *
+ * ```ts
+ * const emailRouting = await EmailRouting("example.com-routing", {
+ *   zone: "example.com",
+ *   enabled: true
+ * });
+ * ```
+ *
+ * @example
+ * ## Use with existing Zone resource
+ *
+ * Configure email routing using an existing Zone resource reference.
+ *
+ * ```ts
+ * const zone = await Zone("my-zone", {
+ *   name: "example.com"
+ * });
+ *
+ * const emailRouting = await EmailRouting("example.com-routing", {
+ *   zone: zone,
+ *   enabled: true,
+ *   skipWizard: true
+ * });
+ * ```
+ */
+export const EmailRouting = Resource(
```

Comment:
This is using deprecated APIs. It should be using https://developers.cloudflare.com/api/resources/email_routing/subresources/dns/methods/create/